boskos/reaper/deployment.yaml

```diff
@@ -14,4 +14,4 @@ spec:
       terminationGracePeriodSeconds: 30
       containers:
       - name: boskos-reaper
-        image: gcr.io/k8s-testimages/reaper:v20170502-1897e866
+        image: gcr.io/k8s-testimages/reaper:v20170710-2501633f
```

Comment:
Hmm, I'm having trouble finding 2501633f in kubernetes/test-infra? Was this commit erased by chance?